src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machines {
+
+    @Override
+    public String doWork() {
+        return "Bulldozer started it's work";
```

Comment:
Like this in all `doWork` and `stopWork` methods:

```suggestion
    @Override
    public void doWork() {
        System.out.println("Bulldozer started it's work");
```